usaspending_api/etl/management/commands/swap_in_new_table.py

```diff
@@ -281,39 +283,50 @@ def swap_index_sql(self, cursor):
         if rename_sql:
             cursor.execute("\n".join(rename_sql))
 
-    @transaction.atomic
-    def swap_table_view_sql(self, cursor):
-        logger.info("Renaming the new and old tables and views")
-        sql_table_template = "ALTER TABLE {old_table_name} RENAME TO {new_table_name};"
-
-        rename_sql = [
-            f"ALTER TABLE {self.temp_table_name} SET SCHEMA {self.curr_schema_name};",
-            sql_table_template.format(
-                old_table_name=self.curr_table_name, new_table_name=f"{self.curr_table_name}{self.old_suffix}"
-            ),
-            sql_table_template.format(old_table_name=self.temp_table_name, new_table_name=f"{self.curr_table_name}"),
-        ]
-
-        sql_view_template = "ALTER {mv_s}VIEW {old_view_name} RENAME TO {new_view_name};"
+    def swap_dependent_views(self, cursor):
```

Comment:
is there a reason why we’re not making `swap_dependent_views` atomic like `swap_table`?

Reply:
Missed the atomic within that function 👍